arch/x86/guest/guest.c

```diff
@@ -181,7 +181,7 @@ uint64_t gva2gpa(struct vm *vm, uint64_t cr3, uint64_t gva)
 	addr = cr3;
 
 	for (level = 3; level >= 0; level--) {
-		addr = addr & IA32E_REF_MASK;
+		addr = addr & CPU_PAGE_MASK;
```

Comment:
This make senses as it is used to walk the page_table for guest.

Reply:
Why we use CPU_PAGE_MASK here? What will happen if XD set? Should we still keep IA32_REF_MASK?

Reply:
@JasonChenCJ ,  This is only used to obtain the corresponding address in gva2hva. 
The paging mechanism can use the CPU_PAGE_MASK macro to return the address for next level.

Reply:
for cr3, here we can use CPU_PAGE_MASK, but next to ‘addr = entry;‘, then here should use IA323_REF_MASK